blog/_posts/2025-02-04-highlights-2024.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: blog-detail
+post-type: blog
+by: The Scala Organization
+title: Scala Highlights from 2024
+---
+
+
+Welcome to the first edition of **Scala Highlights**, a new quarterly newsletter showcasing technical achievements, online resources, and community news.
+
+The newsletter is a joint effort by all the parts of the Scala organization: the Scala Center, LAMP, Akka, and VirtusLab.
```

Comment:
I feel like "all the parts of the Scala organization" might not be the best way to start this awesome report since some organization or non-organization might feel excluded from "all".

How about: The newsletter is a joint effort by the Scala Center, LAMP, Akka, and VirtusLab, the four core organizations involved in the Scala language development.

Hopefully the above conveys the effect that you are going for.